src/lib/components/footer.svelte

```diff
@@ -1,33 +1,38 @@
 <script lang="ts">
   import ThemeSwitcher from './theme-switcher.svelte';
   import PackageVersion from './package-version.svelte';
+  
 </script>
 
 <footer class="mt-auto w-full bg-skin-off-background py-4">
   <div class="mx-auto flex w-[90%] max-w-[1440px] items-center justify-between">
     <ThemeSwitcher />
     <PackageVersion />
-    <ul class="flex list-none items-center justify-center gap-8">
-      <li>
-        <a
+  
+
+      <ul class="flex list-none items-center justify-center gap-8">
+        <li>
+          <a 
+          style:color={hovering ? 'yellow' : 'blue'}
           data-test-id="discord-btn"
-          class="transition duration-200 hover:text-primary-100 active:text-primary-100"
+          class="transition duration-200 hover:text-[#7289da] active:text-primary-100"
           href="http://discord.eddiehub.org"
           target="_blank"
           rel="noreferrer"
           aria-label="Discord"><i class="fa-brands fa-discord fa-xl" /></a
-        >
-      </li>
-      <li>
-        <a
+          >
+        </li>
+        <li>
+          <a
           data-test-id="github-btn"
           target="_blank"
           rel="noreferrer"
-          class="transition duration-200 hover:text-primary-100 active:text-primary-100"
+          class="transition duration-200 hover:text-[#fafafa] active:text-primary-100"
```

Comment:
`#fafafa` is very close to white, we would need different hover effects here for light and dark mode. You can add the one for dark mode with tailwinds `dark:` operator

Reply:
can we add inverting of color on hover: means background black and icon white/#fafafa on hover of github icon?
also any suggestion regarding discord icon?

Reply:
I think the discord icon is fine, only the Github one needs some adjustment depending on light/darkmode, probably something like `dark:hover:text-[#fafafa] hover:text-[#333333]`